x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ilm/WaitForRolloverReadyStep.java

```diff
@@ -35,6 +39,8 @@ public class WaitForRolloverReadyStep extends AsyncWaitStep {
 
     public static final String NAME = "check-rollover-ready";
 
+    public static final long MAX_PRIMARY_SHARD_DOCS_FOR_TSDB = 200_000_000L;
```

Comment:
Since https://github.com/elastic/elasticsearch/pull/94065/commits/53c155b5b9dd813e0200d8d6f9ab7d35e9802f65, this is not TSDB specific.

```suggestion
    public static final long MAX_PRIMARY_SHARD_DOCS = 200_000_000L;
```